Istio Status Conditions (#520)
kyma-project/istio: docs/user/04-00-istio-custom-resource.md

| Parameter            | Type   | Description                                                                                                                        |
|----------------------|--------|------------------------------------------------------------------------------------------------------------------------------------|
| **state** (required) | string | Signifies the current state of the CR. Its value can be either `Ready`, `Processing`, `Error`, `Warning`, or `Deleting`. |

> | **conditions**  | \[\]object | Current conditions state of the CR. |

Reviewer: ```suggestion
| **conditions**  | \[\]object | Represents the current state of the CR's conditions. |
```

| **conditions.&#x200b;lastTransitionTime**  | string | Defines the date of the last condition status change. |
| **conditions.&#x200b;message**  | string | Provides more details about the condition status change. |
| **conditions.&#x200b;reason**  | string | Defines the reason for the condition status change. |

> | **conditions.&#x200b;status** (required) | string | Status of the condition. The value is either `True`, `False`, or `Unknown`. |

Reviewer: ```suggestion
| **conditions.&#x200b;status** (required) | string | Represents the status of the condition. The value is either `True`, `False`, or `Unknown`. |
```

> | **conditions.&#x200b;type**  | string | Short description of the condition. |

Reviewer: ```suggestion
| **conditions.&#x200b;type**  | string | Provides a short description of the condition. |
```
